jenkins/pipelines/cd/tiflow-operator-docker.groovy

```diff
@@ -46,6 +46,7 @@ pipeline {
             agent {
                 kubernetes {
                     yaml podYaml
+                    nodeSelector "kubernetes.io/arch=amd64"
```

Comment:
It seems not nessarary, because docker is multi-platform

Reply:
We prefer to prioritize the use of amd64 nodes because there are fewer arm64 resources available.